tests/lean/string_imp.lean.expected.out

```diff
@@ -3,7 +3,8 @@ string_imp.lean:1:4: error: equation compiler failed (use 'set_option trace.eqn_
 string_imp.lean:5:3: error: invalid pattern: variable, constructor or constant tagged as pattern expected
 string_imp.lean:5:3: error: function expected at
   string_imp.mk
+term has type
+  _
 string_imp.lean:4:4: error: equation compiler failed (use 'set_option trace.eqn_compiler.elim_match true' for additional details)
 string_imp.lean:8:0: error: unknown identifier 'string_imp.cases_on'
-string_imp.lean:8:0: error: function expected at
-  ⁇
+string_imp.lean:7:0: warning: declaration 'h' uses sorry
```

Comment:
@gebner `error_already_reported` seems to fail here. Could it be that this is the first instance of a definition that has a parser error but no elab error?